pytodo/modules.py

```diff
@@ -110,14 +110,32 @@ def clear():
 
 def update_by_index(todos, index):
     """
-    Updates the text of a todo
+    Updates the text and due date of a todo 
     """
     try:
         todo = todos[index]
-        new_text = input("Enter new text: ")
-        collection.find_one_and_update(
-            {"_id": todo["_id"]}, {"$set": {"text": new_text}}
-        )
+        print("Do you want to update the title?: [y/n]")
```

Comment:
Currently it takes the y/n input in the next line. Make sure it is on the same line (that is usually how it is).